### ISSUE TITLE
Video recording not working in chrome docker

### ISSUE THREAD

**Issue by saurabh2k26** [alexschwantes/testcafe-browser-provider-selenium#11]
Suppose i am on mac and want to test my testcafe scripts on linux chrome.
Approach i have followed is to use chrome docker image along with package - testcafe-browser-provider-selenium
It all works fine, but now i need to record video and i am getting error:
`Video recording is not supported by the "selenium:chrome" browser.`

**What is your Test Scenario?**
Run testcafe on chrome docker image and record video successfully.

**What is the Current behavior?**
getting error:
`Video recording is not supported by the "selenium:chrome" browser.`

**What is the Expected behavior?**
Video should be recorded

**What is your web application and your TestCafe test code?**
Precondition: this package should be installed:
`npm install testcafe-browser-provider-selenium
`
Test code (my_first_test.js)

```
fixture `Getting Started`
    .page `http://google.com`;

    test('Sample test', async t => {
        await t
        console.log("webpage opened")
    })
```
DockerFile:

```
FROM node:latest AS node_base

RUN npm install --save @ffmpeg-installer/ffmpeg

FROM selenium/standalone-chrome

USER root

COPY --from=node_base . .

RUN echo "file ends..."
```
Setup container(dockerfile in same folder)

```
docker build -t mychrome:1.6 .
docker run -d -p 4444:4444 -v /dev/shm:/dev/shm mychrome:1.6
```

run testcafe:
`testcafe selenium:chrome my_first_test.js`

Everything works well till now, so lets try to record video:
`testcafe selenium:chrome my_first_test.js --video myvideo`

This is the result:

```
Using locally installed version of TestCafe.
 Running tests in:
 - Chrome 81.0.4044.92 / Linux 0.0

 Getting Started
webpage opened
 ✓ Sample test


 1 passed (2s)

 Warnings (1):
 --
  Video recording is not supported by the "selenium:chrome" browser.
```
I want to have video support. Please check if this is possible.

**Note from testcafe support:**
The selenium browser provider plug-in you mentioned should have this support implemented via the getVideoFrameData method.

**Comment by saurabh2k26** [comment]
Hi team
Any updates on this issue.

**Comment by alexschwantes** [comment]
Hi, thanks for the nicely documented issue :)

Unfortunately selenium doesn't natively support capturing a video of the recording.
So in order to support this feature it would need some custom way to record the videos.

**Comment by alexschwantes** [comment]
Closing as this has been released in v0.5.0
